lib/transport/jsonwire/actions.js

```diff
@@ -153,14 +153,14 @@ module.exports = {
     ///////////////////////////////////////////////////////////
     switchToFrame(frameId) {
       if (frameId === undefined) {
-        return TransportActions.post('/frame');
+        frameId = null;
       }
 
       return TransportActions.post({
         path: '/frame',
         data: {
-          id: String(frameId)
```

Comment:
`String(null)` outputs `'null'`, this is not intended value as JSON should allow `null` as value and it's what the webdriver expects.